.github/workflows/release.yaml

```diff
@@ -27,3 +46,12 @@ jobs:
         uses: helm/chart-releaser-action@v1.5.0
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Push Helm Chart to OCI Registries
+        run: |
+          REPO_OWNER=`echo "${{ github.repository_owner }}" | tr '[:upper:]' '[:lower:]'`
+          # Get packed chart file name
+          PKG_NAME=`ls .cr-release-packages/*.tgz`
+          # pushing chart to OCI registries
+          helm push ${PKG_NAME} oci://registry-1.docker.io/cartesicharts
```

Comment:
I don't recognize this `cartesicharts`.

Reply:
I don't think is wise to have another Docker Hub organization/account just for that.

Why can't we use the current `cartesi` one?

Reply:
Just followed the bitnami charts for it. 

![image](https://github.com/cartesi/helm-charts/assets/51019382/2fdd0ac4-dc6d-40a3-bce0-3b7dac6057ca)

![image](https://github.com/cartesi/helm-charts/assets/51019382/9c86704b-8e99-4ec1-b7e1-03ba443ac2f1)

Reply:
@endersonmaia which one is best for us ? cartesi or cartesicharts ?

Reply:
Fixed !